daml-foundations/daml-ghc/src/DA/Daml/GHC/Compiler/UtilLF.hs

```diff
@@ -160,3 +163,5 @@ ghcTypes = Module
           (ERecCon (TypeConApp (qual (dataTypeCon dataProxy)) [TVar (Tagged "a")]) [])
       }
 
+instance Outputable Expr where
+    ppr = text . renderPretty
```

Comment:
Seems odd giving a UtilLF module an Ouputable instance. Is this really required?

Reply:
It's the `Outputable` instance for `LF.Expr`. The error functions in `Convert.hs` want something `Outputable`. I'll move the instance there.